guides/manual-installation.md

```diff
@@ -5,255 +5,223 @@
+    ```
+
+    > **Note:** SLE15 SP3 requires a provided Prometheus server. The version available through `SUSEConnect --product PackageHub/15.3/x86_64` is outdated and is not compatible with Trento's Prometheus configuration.
+    > Refer to [Option 1: Use existing installation option](#prometheus_install_option_1) for SLE 15 SP3.
+
+    > **Note:** For SLE15 SP4 change the repository to `SUSEConnect --product PackageHub/15.4/x86_64`
+
+1.  Add the Prometheus user/group:
+
+    ```bash
+    groupadd --system prometheus
+    useradd -s /sbin/nologin --system -g prometheus prometheus
+    ```
+
+1.  Install Prometheus using zypper:
+    ```bash
+    zypper in golang-github-prometheus-prometheus
+    ```
+    > **Note:** In case the missing dependency can't be satisfied we have already added the Prometheus user/group. With this, it is safe to proceed with the installation by choosing Solution 2: break golang-github-prometheus-prometheus
+
+1.  <a id="prometheus_trento_configuration"></a>Change Prometheus configuration by replacing the configuration at `/etc/prometheus/prometheus.yml` with: 
+    ```yaml
+    global:
+      scrape_interval: 30s
+      evaluation_interval: 10s
+
+    scrape_configs:
+      - job_name: "http_sd_hosts"
+        honor_timestamps: true
+        scrape_interval: 30s
+        scrape_timeout: 30s
+        scheme: http
+        follow_redirects: true
+        http_sd_configs:
+          - follow_redirects: true
+            refresh_interval: 1m
+            url: http://localhost:4000/api/prometheus/targets
+    ```
+
+    > **Note:** **localhost:4000** in **url: http://localhost:4000/api/prometheus/targets** refers to the location where Trento web service is running.
+
+1.  Enable and start the Prometheus service:
+
+    ```bash
+    systemctl enable --now prometheus
+    ```
+
+1.  If firewalld is running, allow Prometheus to be accessible from Docker and add an exception on firewalld:
+
+    ```bash
+    firewall-cmd --zone=public --add-port=9090/tcp --permanent
+    firewall-cmd --reload
+    ```
+
+### Install PostgreSQL
+This guide was tested with the following PostgreSQL version:
+
+- **13.9 for SP3**
+- **14.10 for SP4**
+- **15.5 for SP5**
+ 
+Using a different version of PostgreSQL may require different steps or configurations, especially when changing the major number. For more details, refer to the official [PostgreSQL documentation](https://www.postgresql.org/docs/).
+
+1. Install PostgreSQL server:
+    ```bash
+    zypper in postgresql-server
+    ```
+
+1. Enable and start PostgreSQL server:
+
+    ```bash
+    systemctl enable --now postgresql
+    ```
+
+#### Configure PostgreSQL
+
+1. Start `psql` with the `postgres` user to open a connection to the database:
+    ```bash
+    su - postgres
+    psql
+    ```
+1. Initialize the databases in `psql` console:
+      ```sql
+      CREATE DATABASE wanda;
+      CREATE DATABASE trento;
+      CREATE DATABASE trento_event_store;
+      ```
+1.  Create the users:
```

Comment:
Be careful, sometimes Markdown requires a new line between the list. Be more generous and introduce some newlines between the list item. :wink: